src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Suppliers/Blocks/form.html.twig

```diff
@@ -0,0 +1,152 @@
+        'label': 'Name'|trans({}, 'Admin.Global'),
+        'help': invalidCatalogNameHint
+      }) }}
+
+      {% set invalidGenericNameHint %}
+        {{ 'Invalid characters:'|trans({}, 'Admin.Notifications.Info') ~ ' <>={}' }}
+      {% endset %}
+
+      {{ ps.form_group_row(supplierForm.description, {}, {
+        'label': 'Description'|trans({}, 'Admin.Global'),
+        'help': 'Will appear in the list of suppliers.'|trans({}, 'Admin.Catalog.Help') ~ invalidCatalogNameHint
+      }) }}
+
+      {{ ps.form_group_row(supplierForm.phone, {}, {
+        'label': 'Home phone'|trans({}, 'Admin.Global'),
+        'hint': 'Phone number for this supplier'|trans({}, 'Admin.Catalog.Help')
+      }) }}
+
+      {{ ps.form_group_row(supplierForm.mobile_phone, {}, {
+        'label': 'Mobile phone'|trans({}, 'Admin.Global'),
+        'hint': 'Mobile phone number for this supplier.'|trans({}, 'Admin.Catalog.Help')
+      }) }}
+
+      {{ ps.form_group_row(supplierForm.address, {}, {
+        'label': 'Address'|trans({}, 'Admin.Global'),
+      }) }}
+
+      {{ ps.form_group_row(supplierForm.address2, {}, {
+        'label': 'Address (2)'|trans({}, 'Admin.Global'),
+      }) }}
+
+      {{ ps.form_group_row(supplierForm.post_code, {}, {
+        'label': 'Zip/postal code'|trans({}, 'Admin.Global'),
+      }) }}
+
+      {{ ps.form_group_row(supplierForm.city, {}, {
+        'label': 'City'|trans({}, 'Admin.Global'),
+      }) }}
+
+      {{ ps.form_group_row(supplierForm.id_country, {'attr': {
+        'class': 'js-supplier-country-select',
+        'data-states-url': path('admin_country_states') }}, {
+        'label': 'Country'|trans({}, 'Admin.Global'),
+      }) }}
+
+      <div class="js-supplier-state">
+        {{ ps.form_group_row(supplierForm.id_state, {}, {
+          'label': 'State'|trans({}, 'Admin.Global'),
+        }) }}
+      </div>
+
+      {{ ps.form_group_row(supplierForm.logo, {}, {
+        'label': 'Logo'|trans({}, 'Admin.Global'),
+        'help': 'Upload a supplier logo from your computer.'|trans({}, 'Admin.Catalog.Help')
+      }) }}
+
+      {% if logoImage is defined and logoImage is not null %}
+        <div class="form-group row">
+          <label class="form-control-label"></label>
+          <div class="col-sm">
+            {% include '@PrestaShop/Admin/Sell/Catalog/Suppliers/logo_image.html.twig' %}
+          </div>
+        </div>
+      {% endif %}
+
+      {{ ps.form_group_row(supplierForm.meta_title, {}, {
+        'label': 'Meta title'|trans({}, 'Admin.Catalog.Feature'),
+        'help': invalidGenericNameHint
+      }) }}
+
+      {{ ps.form_group_row(supplierForm.meta_description, {}, {
+        'label': 'Meta description'|trans({}, 'Admin.Global'),
+        'help': invalidGenericNameHint
+      }) }}
+
+      {% set keywordHint %}
+        {{ 'To add "tags", click inside the field, write something, and then press "Enter".'|trans({}, 'Admin.Catalog.Help') }}
```

Comment:
Following what has been said in https://github.com/PrestaShop/PrestaShop/pull/10408#discussion_r221600791, please write _To add tags, click in the field, write something, and then press the "Enter" key._ and localize it in _Admin.Shopparameters.Help_, thank you. :-)